### PR TITLE
Fix bad markdown file

### PR DESCRIPTION
**Pull request by hoxbro**
Fixes the problem seen here: https://github.com/holoviz/panel/issues/7119#issuecomment-2307673850

I will run the docs build to verify. https://github.com/holoviz/panel/actions/runs/10538265033

### CODE REVIEW

File: doc/how_to/custom_components/esm/build.md [hunk 1]
```diff
@@ -100,7 +100,8 @@ To recap here are the three files that we need:
 
 ::::{tab-set}
 
-:::tab-item package.json
+:::{tab-item} package.json
```
Comment (hoxbro) [inline doc/how_to/custom_components/esm/build.md:103]: This is the main problem

**Comment by hoxbro** [comment]
We should have a way to test markdown files so an error like this does not happen again.

**Review by philippjfr — APPROVED** [review]

**Comment by philippjfr** [comment]
Good catch, thanks!

**Comment by philippjfr** [comment]
>We should have a way to test markdown files so an error like this does not happen again.

Agreed, guess we could just run MyST parser on all markdown files.

**Comment by philippjfr** [comment]
Will merge this for now.

**Pull request merged**